### ISSUE TITLE
search_primer_pair finding reverse complement sequences

### ISSUE THREAD

**Issue by analaca** [MVesuviusC/primerTree#39]
Hi,

I just tried the function search_primer_pair. It is supposed to search for the forward primer by 5’-3’ on plus strand and for the reverse primer by 5’-3’ on minus strand. However, I got results for sequences that were reversed on GenBank, hence the outliers in the tree below.

![Tridac16S_insilicoNCBI](https://user-images.githubusercontent.com/63670646/79264418-662a9300-7e49-11ea-9c72-cc8e9152de67.jpeg)

`search_primer_pair(
GTGTTAAACCTTAAAGTAGCG, 
ATCCAACATCGAGGTCAC,
name = NULL,  
num_aligns = 500, 
  num_permutations = 25,
  simplify = TRUE,
  clustal_options = list(exec = "clustalo", quiet = TRUE, original.ordering = TRUE),
  distance_options = list(model = "N", pairwise.deletion = T),
  api_key = "xxxxxxxx",
  .parallel = FALSE,
  .progress = "none"
)`

It's actually a good thing that the function was able to find them, but it would be great if it could detect this issue and reverse complement them in order to have them in the right direction in the alignment.

Thanks for this package. It's going to be very useful for my research.
Cheers

**Comment by MVesuviusC** [comment]
Yeah, I've seen this before as well. it also can mess up the calc_rank_dist_ave numbers since the genetic distance between sequences within a species/genera can get overestimated. 

I don't know that clustal can fix this automatically, but if we switched to mafft and used "--adjustdirectionaccurately" it would solve this issue.
